minifi-nar-bundles/minifi-framework-bundle/minifi-framework/minifi-runtime/pom.xml

```diff
@@ -40,6 +40,24 @@ limitations under the License.
             <groupId>org.apache.nifi.minifi</groupId>
             <artifactId>minifi-api</artifactId>
         </dependency>
+        <dependency>
+            <groupId>ro.pippo</groupId>
+            <artifactId>pippo-core</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>ro.pippo</groupId>
+            <artifactId>pippo-tomcat</artifactId>
```

Comment:
That works for me, the only change that would be impact the user would be the changing of a couple config properties (like a removal of the base folder)